e2e/suite_test.go

```diff
@@ -173,7 +193,25 @@ func bootstrapVMSS(ctx context.Context, t *testing.T, r *mrand.Rand, opts *scena
 	return vmssName, vmssModel, cleanupVMSS, nil
 }
 
-func getNodeBootstrapping(ctx context.Context, nbc *datamodel.NodeBootstrappingConfiguration) (*datamodel.NodeBootstrapping, error) {
+func getNodeBootstrappingForCoverage(ctx context.Context, nbc *datamodel.NodeBootstrappingConfiguration) (*datamodel.NodeBootstrapping, error) {
```

Comment:
I'd also move getNodeBootstrapping{ForCoverage/ForValidation} to nbc/nodebootstrapping.go, then maybe have a function which wraps the 2 called getNodeBootstrappingConfiguration that calls one or the other based on `e2eMode` - that way we don't have to plumb around a function ptr

Reply:
Moved to another file